vpr/src/place/place_util.h

```diff
@@ -5,20 +5,56 @@
  */
 
 #pragma once
+#include "vpr_types.h"
 
-struct t_placer_costs {
-    //Although we do nost cost calculations with float's we
-    //use doubles for the accumulated costs to avoid round-off,
-    //particularly on large designs where the magnitude of a single
-    //move's delta cost is small compared to the overall cost.
+/**
+ * @brief Data structure that stores different cost values in the placer.
+ *
+ * Although we do cost calculations with float values, we use doubles
+ * for the accumulated costs to avoid round-off, particularly on large
+ * designs where the magnitude of a single move's delta cost is small
+ * compared to the overall cost.
+ *
+ * The cost normalization factors are updated upon every temperature change
+ * in the outer_loop_update_timing_info routine. They are the multiplicative
+ * inverses of their respective cost values when the routine is called. They
+ * serve to normalize the trade-off between timing and wirelength (bb).
```

Comment:
Explain why we store the inverses: speed (since we use them a lot and divides are slow).  Could also do an experiment to make sure they do indeed speed things up (not sure I ever tested, just coded them in advance since I know divides are slow).